### ISSUE TITLE
Tags not showing correctly while editing from posts list

### ISSUE THREAD

**Issue by Cadiducho** [flarum/core#2051]
## Bug Report

**Current Behavior**
I'm getting an strange design error while I try to edit tags of a post from the main list

**Steps to Reproduce**
1. Go to your main discussion list
2. Click on the three dots
3. Click on "Edit Tags"

**Expected Behavior**
Get the current tags list, not to see them dissapear

**Screenshots**
![First](https://i.imgur.com/BE4xExx.png)
![Second](https://i.imgur.com/F6OvAIP.png)
![Tags exists in html](https://i.imgur.com/l9m0ibZ.png)

**Environment**
- Flarum version: 0.1.0-beta.12 (also in beta.11)
- Website URL: https://foro.edoras.es
- Webserver: CaddyServer
- Hosting environment: vps
- PHP version: 7.3.15
- Browser: Chrome 80, Edge 80. Also without extensions and incognito

```
Flarum core 0.1.0-beta.12
PHP version: 7.3.15
Loaded extensions: Core, date, libxml, openssl, pcre, zlib, filter, hash, pcntl, readline, Reflection, SPL, session, standard, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, json, ldap, exif, mysqlnd, odbc, PDO, Phar, posix, shmop, SimpleXML, soap, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlwriter, xsl, mcrypt, mysqli, pdo_mysql, PDO_ODBC, pdo_sqlite, wddx, xmlreader, xmlrpc, ssh2, Zend OPcache
+---------------------------------+-----------------+--------+
| Flarum Extensions               |                 |        |
+---------------------------------+-----------------+--------+
| ID                              | Version         | Commit |
+---------------------------------+-----------------+--------+
| flarum-approval                 | v0.1.0-beta.12  |        |
| flarum-bbcode                   | v0.1.0-beta.12  |        |
| flarum-flags                    | v0.1.0-beta.12  |        |
| flarum-likes                    | v0.1.0-beta.12  |        |
| flarum-mentions                 | v0.1.0-beta.12  |        |
| flarum-sticky                   | v0.1.0-beta.12  |        |
| flarum-tags                     | v0.1.0-beta.12  |        |
| flarum-subscriptions            | v0.1.0-beta.12  |        |
| flarum-markdown                 | v0.1.0-beta.12  |        |
| flarum-suspend                  | v0.1.0-beta.12  |        |
| flarum-lock                     | v0.1.0-beta.12  |        |
| flarum-statistics               | 0.8.0           |        |
| michaelbelgium-discussion-views | v0.1.0-beta.8.3 |        |
| flarum-emoji                    | v0.1.0-beta.12  |        |
| fof-linguist                    | 0.4.2           |        |
| fof-links                       | 0.2.1           |        |
| fof-pages                       | 0.2.7           |        |
| fof-polls                       | 0.1.1           |        |
| fof-prevent-necrobumping        | 0.2.0           |        |
| fof-reactions                   | 0.1.2           |        |
| fof-split                       | 0.4.3           |        |
| fof-user-bio                    | 0.1.3           |        |
| flarumes-spanish                | v0.1.0-beta.7   |        |
| zerosonesfun-tooltip            | 1.1             |        |
| zerosonesfun-spoiler            | 1.2             |        |
| zerosonesfun-pop                | 1.0             |        |
| zerosonesfun-font               | 1.1             |        |
| zerosonesfun-chat               | 1.0             |        |
| zerosonesfun-blur               | 1.1             |        |
| zerosonesfun-bbbbcode           | 3.3             |        |
| zerosonesfun-bbcode-audio       | 1.8             |        |
| zerosonesfun-action             | 1.0             |        |
| zerosonesfun-accordion          | 2.4             |        |
+---------------------------------+-----------------+--------+
Base URL: https://foro.edoras.es
Installation path: /var/www/foroedoras
Debug mode: off
```

**Additional Context**
This happens in any post, not only in those who have multiple tags

I recorded a video, just logged in with Edge
https://streamable.com/1ipg1

**Comment by matteocontrini** [comment]
#2030

**Comment by matteocontrini** [comment]
@askvortsov1 sorry for tagging. This is a duplicate of #2030, could it be closed, perhaps? #2040 is also related, it seems.

**Comment by askvortsov1** [comment]
Yep, thanks for letting me know! Cadudicho, thank you for the report though!
